lib/puppet/provider/sensuctl.rb

```diff
@@ -30,7 +30,8 @@ def convert_boolean_property_value(value)
   def self.sensuctl_list(command)
     args = [command]
     args << 'list'
-    if ! ['organization','role','user'].include?(command)
+    # TODO: Remove extension once resolved: https://github.com/sensu/sensu-go/issues/1767
+    if ! ['extension','organization','role','user'].include?(command)
```

Comment:
This is a temporary workaround until https://github.com/sensu/sensu-go/issues/1767 is resolved.